common/config/app.py

```diff
@@ -134,6 +134,11 @@
 )
 connector_api_port = _config("CONNECTOR_API_PORT", default=5000)
 
+internal_watson_extension_base_url = _config(
+    "INTERNAL_WATSON_EXTENSION_BASE_URL", default="/internal/api/v1"
```

Comment:
we can probably go ahead and mount this as /api/v1 (or /api/assistant-extension/v1)